github-action/src/main.ts

```diff
@@ -46,6 +48,17 @@ export async function runMain(): Promise<void> {
 		);
 		const userDataFolder: string = core.getInput('userDataFolder');
 
+		if (platform) {
+			const skopeoInstalled = await isSkopeoInstalled();
+			if (!skopeoInstalled) {
+				core.warning(
```

Comment:
Is there a way to avoid the dependency on an additional CLI?

Reply:
So, yes and no. The docker cli can do a build and push in one step, but then the ability to run the intermediate runCmd is lost. (There's no way to just push with the docker cli that I was able to find). I figured there was value in still being able to use runCmd on multi-platform builds, which is why I went with the strategy of building to an OCI tar, running the runCmd, and then, if it succeeds, doing the push via skopeo.

FWIW, skopeo is included by default in all runner images.

Reply:
Hi @chrmarti, sorry for the ping but just wanted to make sure you saw my response. Thanks!

Reply:
Thanks for the ping and the detailed response! If it's installed by default, we're fine for now. Thanks!